### ISSUE TITLE
Date of report is a required field and does not allow the user to choose unknown

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#1471]
Some times, the date of report is missing on the case forms. The variable should stay as a required field and also allow the user to choose unknown as an option. The current situation leads to false date reading because the user is always forced to chose a data even  when the date is missing.
This should be the case fro most required fields but not for all. I think we should define the variables that this should apply.

**Comment by Iheanacho2027** [comment]
Yes, its a very good point. Also the idea of the system selecting the current date as the Date of report if the user doesnt select a date is unideal. Like @bernardsilenou said, the unknown variable should be there, and if the user doesnt know the date, he should select unknown

**Comment by MateStrysewskeSym** [comment]
@bernardsilenou @Iheanacho2027 We need to discuss this in our next review because the date of report currently is used in all case calculations that are based on dates.

**Comment by MateStrysewskeSym** [comment]
Gérard agreed that we need to keep this a required field, but suggested renaming it to something like "Date of earliest local report" or "Earliest local reporting date".
